client/src/pages/Home/Home.jsx

```diff
@@ -37,34 +39,66 @@ const useStyles = makeStyles((theme) => ({
 const Home = () => {
   const classes = useStyles();
   const [polls, setPolls] = useState([]);
+  const [friendLists, setFriendLists] = useState([]);
   const { data, isLoading, isFetching } = useQuery('polls', getPolls);
+  const [friendsData, setFriendsData] = useState([]);
 
-  useEffect(() => {
-    setPolls(data);
-  }, [data]);
+  // useEffect(() => {}, [data]);
 
   const handlePolls = (info) => {
     setPolls(info);
   };
 
+  //Helper function for retrieving the friendlists
+  const fetchData = async () => {
+    const res = await getFriendLists();
+    const getFriendsIds = await getFriends();
+    console.log('friend ids', getFriendsIds);
+
+    // const friendInfo = getFriendsIds.map((info) => {
+    //   [getFriendInfo(info)];
+    // });
+    // console.log('friend info', friendInfo);
+    setFriendLists(res);
+  };
+
+  // const { friendListData } = useQuery('friendlists', getFriendLists);
+
+  useEffect(() => {
+    setPolls(data);
+    fetchData();
+    console.log(data);
+    // console.log('friendListData', friendListData);
+  }, []);
+
+  const handleFriendLists = (info) => {
+    setFriendLists(info);
+  };
+
   return (
     <Box className={classes.mainContainer}>
       <Box className={classes.left}>
         <Friends friendList={Array(10).fill({ name: 'demo' })} />
       </Box>
       <Box className={classes.right}>
         <Polls handlePolls={handlePolls} listOfPolls={polls} className={classes.polls} />
-        <FriendList
+        {/* <FriendList
```

Comment:
if you dont need comment .. lets do one thing and its applicable for all of us... let remove the comments from our codebase @SsaaaadD2  @TheAllen